docs/Home.md

```diff
@@ -1,3 +1,50 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Ex: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria:
+
+How do we decide if a feature is **essential**? 👍
+
+- Our team requires it to maintain code quality.
+- Our team requires it to maintain a good developer experience.
+- Our team requires it to increase the productivity of the team.
+- It is always used or configured in the projects we've worked on.
+- It is recommended to use or configure based on Google's standards.
+
+How do we decide if a feature is **optional**? 🚩
+
+- It is regularly used or configured in projects we've worked on.
+- It is regularly used or configured as an alternative to the default choice.
+
+How do we decide if a feature is **rejected**? 👎
+
+- It is barely used or configured in projects we've worked on.
+- It is barely used or configured as an alternative to the default choice.
```

Comment:
same here, we need to be more directive and succinct: Components that we don't apply to most of the project, come as a specific project requirement. Right? 🤔

Reply:
and the idea here is to unify the ways/components our team should be familiar with and spend effort on maintaining, so by rejecting the uncommon idea, we can have more focus on the real matter.